releasenotes/notes/0.11/full-rary-tree-generator-4c533fd73dbd3e48.yaml

```
@@ -2,7 +2,7 @@
 features:
   - |
     Added a new function, :func:`~retworkx.generators.full_rary_tree` that adds
-    support for generating a full r-ary tree of n nodes. For example:
+    support for generating a full r-ary tree of :math:`n` nodes. For example:
```

If you make the `n` in the maths font, you probably want the `r` as well:
```suggestion
    support for generating a full :math:`r`\ -ary tree of :math:`n` nodes. For example:
```